bindsnet/pipeline/__init__.py

```diff
@@ -199,11 +239,13 @@ def plot_data(self) -> None:
         # Initialize plots
         if self.s_ims is None and self.s_axes is None and self.v_ims is None and self.v_axes is None:
             self.s_ims, self.s_axes = plot_spikes(self.spike_record)
-            self.v_ims, self.v_axes = plot_voltages(self.voltage_record)
+            self.v_ims, self.v_axes = plot_voltages(self.voltage_record,
+                    plot_type=self.plot_type, threshold=self.threshold_value)
         else:
             # Update the plots dynamically
             self.s_ims, self.s_axes = plot_spikes(self.spike_record, ims=self.s_ims, axes=self.s_axes)
-            self.v_ims, self.v_axes = plot_voltages(self.voltage_record, ims=self.v_ims, axes=self.v_axes)
+            self.v_ims, self.v_axes = plot_voltages(self.voltage_record, ims=self.v_ims,
```

Comment:
I noticed you specify `plot_type='line'`, but it seems like you haven't changed `plot_voltages` to implement line plots. Am I missing something?

Reply:
Actually I've changed plot_voltage function and added 'line' plot_type. I think it is not shown because there was merge conflict in plotting.py. And I also found that the conflict was kind of weird because it says every single line in the file are conflicting while they actually don't. I think it is probably related to using different new line character between commits.

Reply:
Ah, got it.